spec/jobs/renewal_job_spec.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
```

Comment:
Filename needs to be `renew_checkout_job_spec.rb` right?

Reply:
ahhh yes! thanks!

Reply:
fixed